.github/workflows/functional-test.yaml

```diff
@@ -94,8 +105,7 @@ jobs:
             echo "CHECKOUT_REPO=${{ github.repository }}" >> $GITHUB_ENV
             echo "CHECKOUT_REF=${{ github.ref }}" >> $GITHUB_ENV
             echo "PR_NUMBER=${{ github.event.pull_request.number }}" >> $GITHUB_ENV
-      - name: Set up checkout target (repository_dispatch from /ok-to-test)
-        if: github.event_name == 'repository_dispatch'
+      - name: Set up checkout target
```

Comment:
It looks like all of the code is here to handle dispatch from `ok-to-test`, it just got renamed.

Should this be a normal `actions/checkout`?